### PR TITLE
changed display of params

### PR DESCRIPTION
**Pull request by matthdsm**
Changed display of params to radio button to increase the visibility of possible option, current version only shows an empty field with a "check/uncheck" box.

### CODE REVIEW

File: tools/table_annovar/table_annovar.xml [hunk 1]
```diff
@@ -63,21 +63,21 @@
         <validator type="unspecified_build" />
     </param>
 
-    <param name="gene_anns" type="select" multiple="True" optional="True" label="Gene Annotations" help="" >
+    <param name="gene_anns" type="select" display="radio" multiple="True" optional="True" label="Gene Annotations" help="" >
```
Comment (nsoranzo) [inline tools/table_annovar/table_annovar.xml]: I think `display="checkboxes"` would be better. Radio buttons are normally used for mutually exclusive options, while here multiple selections are explicitly allowed: https://www.nngroup.com/articles/checkboxes-vs-radio-buttons/

Comment (matthdsm) [inline tools/table_annovar/table_annovar.xml]: I agree, would be more clear that way. Changed it!

Thanks for the input!
M

**Comment by matthdsm** [comment]
On an unrelated note,
I'm trying to avoid to have to edit PATH of my galaxy user, but I can't seem to figure out where to put the env.sh file to set the environmental value.
Any help or resource? I'm a bit confused by the wiki entry at the [wiki](https://wiki.galaxyproject.org/Admin/Config/ToolDependencies)

Thanks
M

**Comment by nsoranzo** [comment]
@matthdsm The following path may work: `YOUR_TOOL_DEP_DIR/environment_settings/SCRIPT_PATH/devteam/table_annovar/08b003ee9db7/env.sh` , but the revision (08b003ee9db7) will change at every tool repository update.

**Comment by matthdsm** [comment]
Thanks! I forgot I could find those files there!
M

**Pull request merged**